### PR TITLE
Proposed Standard Change regarding Return Type Hinting

### PR DESCRIPTION
**Pull request by calevans**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,4 +59,12 @@
 - Use paren bracket style delimiters for preg_* functions and the like `preg_match('(^\d+)', $text)`
 - No trailing whitespace on any line
 - Every file must end with a single new line character
+
+- All PHP functions should have a return typehint if it returns a value.
+ 1. If a return typehint causes the line to exceed the maximum line length then the colon indicating a return type hint should stay with the closing parenthesis.
```
Comment (robertology) [inline README.md]: is this really saying that the colon always stays with that paran?

File: README.md [hunk 2]
```diff
@@ -59,4 +59,12 @@
...
+ 1. If a function's arguments fit on a single line, the typehint should be on the next line and indented.
+ 1. If a function's arguments do not fit on a single line, the typehint should follow the closing parenthesis.
+ 1. In both cases 2 & 3, the opening brace should stay with the typehint.
```
Comment (robertology) [inline README.md]: is there a case where it doesn't?

File: README.md [hunk 3]
```diff
@@ -59,4 +59,12 @@
 - Use paren bracket style delimiters for preg_* functions and the like `preg_match('(^\d+)', $text)`
 - No trailing whitespace on any line
 - Every file must end with a single new line character
+
+- All PHP functions should have a return typehint if it returns a value.
+ 1. If a return typehint causes the line to exceed the maximum line length then the colon indicating a return type hint should stay with the closing parenthesis.
+ 1. If a function's arguments fit on a single line, the typehint should be on the next line and indented.
```
Comment (robertology) [inline README.md]: is this only when the line is too long?

Comment (calevans) [reply README.md]: Yes

File: README.md (hunk 2 above)
Comment (calevans) [reply README.md]: No. just making it explicit what is expected.

File: README.md (hunk 1 above)
Comment (calevans) [reply README.md]: Yes

File: README.md (hunk 3 above)
Comment (adrian-enspired) [reply README.md]: maybe should be

> If a function's arguments fit on a single line, but the typehint does not, then the typehint should be on the next line and indented.

**Pull request closed without merging**